extensions/ipscan/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ipscan",
+  "title": "IPScan",
+  "description": "Scan IP addresses and ports directly from Raycast",
+  "icon": "extension-icon.png",
```

Comment:
You just referred to the wrong image. Here means reading image assets from the `assets` folder, which is the Raycast extension placeholder icon file. You may need to move your `extension-icon.png` into `asset` to fix it.